fv3core/Makefile

```diff
@@ -1,9 +1,6 @@
 include ../docker/Makefile.image_names
 
 GCR_URL = us.gcr.io/vcm-ml
-REGRESSION_DATA_STORAGE_BUCKET = gs://vcm-fv3gfs-serialized-regression-data
-EXPERIMENT ?=c12_6ranks_standard
-FORTRAN_SERIALIZED_DATA_VERSION=8.1.1
 SHELL=/bin/bash
 CWD=$(shell pwd)
 PULL ?=True
```

Comment:
I think we don't need this anymore? Not sure if it was related to pulling data.

Reply:
Not needed anymore, was related to pulling images when the environment was very expensive to build. It's now a lot faster to build so there are no remote images to pull e.g. with built mpich and serialbox.